.github/workflows/ci.yml

```diff
@@ -12,15 +12,13 @@ permissions:
 jobs:
   test:
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python 3.9
-        uses: actions/setup-python@v3
+      - uses: actions/setup-python@v3
```

Comment:
Thoughts on keeping/removing the `name`? I left it in so that it's obvious from reading the CI logs what version of Python is being used, especially if it gets bumped later, but maybe it won't be useful?

Reply:
Figured it'd be a bother to maintain that info in multiple places. Could maybe use `env`/`matrix`, though?

Reply:
Hmm, yeah, probably a good idea to test with 3.10 before Talon starts using 3.10 as well.